portfolio/src/main/webapp/index.html

```diff
@@ -50,6 +50,7 @@ <h1><span class="text-emphasis">Antonio</span>'s Portfolio</h1>
           </p>
         </div>
       </div>
+      <button type="button" class="btn-primary btn-lg" onclick="setRandomBackgroundColor()">Change the background</button>
```

Comment:
I'd recommend keeping Javascript within script.js: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Try using [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) to add an onclick listener to this button.

Reply:
Added! Struggled a bit with figuring out the correct code - didn't know I needed to use an anonymous function